### ISSUE TITLE
Improve UI of the web application

### ISSUE THREAD

**Issue by aapJoysakoi** [princekhunt/privateping#56]
**Is your feature request related to a problem? Please describe.**
The front end appears to be very basic

**Describe the solution you'd like**
A user friendly and eye-catchy yet classy look of the website 
**Describe alternatives you've considered**
A change in color palette.

**Comment by aapJoysakoi** [comment]
please assign me under SSOC

**Comment by princekhunt** [comment]
Go on.

**Comment by princekhunt** [comment]
Are you still on it ?

**Comment by aapJoysakoi** [comment]
Yaaa, I am. (I’ll create a pr by the end of this weekend for sure)

**Comment by princekhunt** [comment]
Are you still on it ?

**Comment by aapJoysakoi** [comment]
Yes, so sorry for the delay. I’m using react for the first time so it’s taking longer than expected.

**Comment by princekhunt** [comment]
PrivatePing does not use react.

**Comment by aapJoysakoi** [comment]
I am using react for the front end.

**Comment by princekhunt** [comment]
We're using simple css and javascript.
